bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/CallOptionsFactory.java

```diff
@@ -112,6 +166,24 @@ private int getRequestTimeout(Object request) {
       }
     }
 
+    /**
```

Comment:
I think we should centralize the deadline merging logic to above and make this a simple getter for attempt deadlines